Hymns/Display/DisplayHymnBottomBar.swift

```diff
@@ -30,16 +31,24 @@ struct DisplayHymnBottomBar: View {
         HStack(spacing: 0) {
             Group {
                 Spacer()
-                Button(action: {}, label: {
-                    BottomBarLabel(imageName: "square.and.arrow.up")
-                })
+                    Button(action: {
+                        self.showShareSheet = true
+                    }, label: {
+                        BottomBarLabel(imageName: "square.and.arrow.up")
+                    })
+                .sheet(isPresented: $showShareSheet) {
+                    ShareSheet(activityItems: [self.viewModel.sharablelLyrics])
+                }
+
                 Spacer()
             }
             Group {
                 Button(action: {self.tabPresented = .fontSize}, label: {
                     BottomBarLabel(imageName: "textformat.size")
                 })
                 Spacer()
+            }.onAppear {
+                self.viewModel.fetchLyrics()
```

Comment:
lol it kind of looks like you just picked a random Group to stick this onAppear{} in.

I would do it on the HStack, just to keep all this stuff in one place. Besides, if for some reason in the future we don't want to show font size based on some param, then this will break. We can pretty much guarantee that the HStack will always be shown, so it's safer to put it there

Reply:
Lol yeah, my bad haha I didn't mean to leave it here. I was just testing something real fast and threw it on here and forgot to move it when it actually ended up working.